specification/sql/resource-manager/readme.md

```diff
@@ -1174,7 +1174,7 @@ input-file:
 - ./Microsoft.Sql/preview/2020-11-01-preview/ElasticPoolOperations.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/EncryptionProtectors.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/FirewallRules.json
```

Comment:
Please also modify Line 27 tag to "package-composite-v5"


![image](https://github.com/Azure/azure-rest-api-specs/assets/3658582/b30f9338-725c-4e06-a5fe-4f180e04578d)

Reply:
About the revert and corresponding failed pipeline, please check my comment in another PR: https://github.com/Azure/azure-rest-api-specs/pull/26683#discussion_r1391989909

You could either add it or wait https://github.com/Azure/azure-rest-api-specs/pull/26683#discussion_r1391989909 merge.